src/bootstrap/index.ts

```diff
@@ -80,7 +80,7 @@ const createListener = (
 }
 
 const bootstrapServer = (app: Express) => {
-  const port = normalizePort(process.env.PORT || "8081") // TODO: PORT is not in env example
+  const port = normalizePort(process.env.PORT || "8081")
```

Comment:
hmm is there a reason why we dont use convict for this?

Reply:
fixed